apps/e2e/ag-grid-storybook/src/app/data-entry-grid/data-entry-grid.component.ts

```diff
@@ -69,7 +69,7 @@ export class DataEntryGridComponent
   public ready = new BehaviorSubject(false);
   public skyTheme: SkyThemeSettings;
 
-  readonly #gridsReady = new Map<string, Observable<boolean>>();
+  #gridsReady: { [_: string]: Observable<boolean> } = {};
```

Comment:
Why the switch away from `Map`? Seems like the `Map` has a cleaner implementation.

Reply:
I was working through an issue that ended up not being related to `Map`. Switched back to using `Map`.